Finding.Object.Game.html

```diff
@@ -0,0 +1,261 @@
+  height: 25px;
+  border-radius: 10px;
+}
+.winner { 
+    background-image: url('http://www.pngall.com/wp-content/uploads/2016/04/Winner-PNG-File.png');
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center; 
+}
+<!---.monster {
+  background-image: url('http://vignette4.wikia.nocookie.net/looneytunes/images/3/34/Sylvester_Cat_Cartoon-psd12937.png');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  right: 40px;
+  background-position: 200px 100px;
+  top: 100px;
+  background-size: 150px;
+}-->
+</style>
+<script type="text/javascript">
+    var nbObject = 0;
+    var score=0;
+	var won = false;
+	var hidden_objects = [];
+	var start_time;
+
+    function init() {
+      document.onkeydown = onKeyPress;
+      var score_elt = document.getElementById("Score");
+	  score_elt.setAttribute("value","0");
+      nbObjects = document.getElementById("NbObjects").value;
+      createHiddenObjects(nbObjects);
+      score = 0;
+      start_time = (new Date()).getTime() / 1000;
+      score_elt.innerHTML = 'Score  = ' + score;
+      
+    }
+    
+    function NbObjectUpdate() { // When user change the Number of Objects field
+      nbObjects = document.getElementById("NbObjects").value;
+	  createHiddenObjects(nbObjects);
+	  start_time = (new Date()).getTime() / 1000;
+      var y = Number(document.getElementById("y").value);
+      var x = Number(document.getElementById("x").value);
+	  var closest = find_closest(x,y);
+	  document.getElementById("distanceMeter").setAttribute("value", closest.distance);
+
+    }
+    
+    function onKeyPress(event) {
+	  if (won) return;
+      var y = Number(document.getElementById("y").value);
+      var x = Number(document.getElementById("x").value);
+	  switch (event.keyIdentifier) {
+	    case "Up":
+          y = y + 1;
+          document.getElementById("y").value = y;
+		  break;
+		case "Down":
+          y = y - 1;
+          document.getElementById("y").value = y;
+		  break;  
+		case "Right":
+          x = x + 1;
+          document.getElementById("x").value = x;
+		  break;  
+		case "Left":
+          x = x - 1;
+          document.getElementById("x").value = x;
+		  break;  
+	  }
+	  var closest = find_closest(x,y);
+	  document.getElementById("distanceMeter").setAttribute("value", closest.distance);
+	  if (closest.distance == 0) {
+	    var score_elt = document.getElementById("Score");
+	    var score = Number(score_elt.getAttribute("value")) + 1;
+		score_elt.setAttribute("value", score);
+		score_elt.innerHTML = 'Score  = ' + score;
+		remove_hidden_object(closest.index);
+		if (hidden_objects.length == 0 ) you_won();
+	  }
+    }
+
+    function createHiddenObjects(nbObjects) {
+	   hidden_objects.length = 0;
+       for (var i=0; i< nbObjects; i++) {
+	     hidden_objects.push({x: Math.floor(Math.random() * 100), 
+		                      y: Math.floor(Math.random() * 100)});
+	   }
+    }
+	
+	function find_closest (x, y) {
```

Comment:
Nice clean function here, good name, no mutation (modifying) of external state, a single and clear responsibility. It does access `hidden_objects` directly but that is consistent with the overall approach.